### DISCUSSION TITLE
Time Series Panel - hover tooltip sort by value and option top x values

### DISCUSSION THREAD

**Discussion by mhollstein** [grafana/grafana#38048] [Dashboards, Panels & Visualizations]
In the new time series panel the hover tooltip can not be sorted like in the old panel. There should be an option to sort it by value.
In addition there should be an option to display only the top x values to minimize long lits.

![Timeseries - hover tooltip sort](https://user-images.githubusercontent.com/56024424/130010097-e1342b05-b3f2-4cac-a6a7-d0e04a794f8a.gif)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
